Example/CardScan/ViewController.swift

```diff
@@ -10,7 +10,8 @@ import UIKit
 import CardScan
 
 class ViewController: UIViewController, ScanEvents, ScanDelegate, FullScanStringsDataSource, TestingImageDataSource {
-
+    var imageIsFullScreen: Bool? = nil
```

Comment:
can we _not_ define this? Our goal is to make sure that the people who are already using this don't need to change their code, so ideally we wouldn't need to make any changes to this function. It might not be possible, but we should try.